src/hotspot/cpu/ppc/macroAssembler_ppc.cpp

```diff
@@ -1185,7 +1185,9 @@ void MacroAssembler::post_call_nop() {
   if (!Continuations::enabled()) {
     return;
   }
+  PostCallNopCounter nopCounter(this);
```

Comment:
Did you forgot to change it to `InlineSkippedInstructionsCounter`?

Reply:
I forgot other thing, but forgot. fixing.